client/src/pages/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,122 @@
+import { useState, useEffect } from 'react';
+import { Notification } from '../../interface/User';
+import { getNotification, deleteNotification } from '../../helpers/APICalls/notification';
+import useStyles from './useStyles';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Avatar from '@material-ui/core/Avatar';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import AuthHeader from '../../components/AuthHeader/AuthHeader';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+
+export default function Notifications(): JSX.Element {
+  const [notifications, setNotifications] = useState<Notification[]>([]);
+  const [delResponse, setdelResponse] = useState<number | null>();
+  const { loggedInUser } = useAuth();
+  const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  useEffect(() => {
```

Comment:
We are fetching all the notifications each time this component mounts, just as above. Please see comment above for solution.

Reply:
used context in the recent commit